### ISSUE TITLE
Save images locally to site build

### ISSUE THREAD

**Issue by ThisIsMissEm** [Unobvious-Technology/gatsby-plugin-source-amazon-wishlist#1]
Currently the `image` field just gives the remote amazon CDN url for an image, it'd be great to save these locally to the build and not hotlink them. I know this can be done, I just don't know right now how to. I've already added some code that was attempting to do this, but then I gave up.

**Comment by pieh** [comment]
Check `createRemoteFileNode` helper function exported from `gatsby-source-filesystem` package - it will download asset and create `File` node for it - then you can either copy file as is or use image processing with `gatsby-transformer-sharp` and `gatsby-plugin-sharp`

Docs: https://www.gatsbyjs.org/packages/gatsby-source-filesystem/#createremotefilenode

**Comment by ThisIsMissEm** [comment]
Do I just assign the File node to the image field? It was the assignment I couldn’t get working

**Comment by ThisIsMissEm** [comment]
This has to be done before I call createNode, correct? The documentation isn’t particularly clear.

**Comment by pieh** [comment]
Yes, before `createNode`. Nodes are stored in redux store and you technically can modify them but this might cause unforeseen issues if you do that
